spec/support/features.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.configure do |configure|
```

Comment:
Might be able to stub here rather than altering the global state. This way is probably fine too but I wonder if there are errors that could occur in `example.run` that would stop the reset happening.

Reply:
I was worried about that too, it turns out that the code after `example.run` still executes even if an expectation fails, or worse, an exception is raised. Which is why I thought an `ensure` block wasn't necessary here.

Also makes sense since this is likely the same mechanism used by mocking libraries. This behaviour is even specced out:

https://relishapp.com/rspec/rspec-core/v/3-7/docs/hooks/around-hooks#an-around-hook-continues-to-run-even-if-the-example-throws-an-exception

Reply:
👍